front_end/panels/ai_chat/ui/CustomProviderDialog.ts

```
@@ -0,0 +1,540 @@
+// Copyright 2025 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import * as UI from '../../../ui/legacy/legacy.js';
+import { CustomProviderManager } from '../core/CustomProviderManager.js';
+import type { CustomProviderConfig } from '../core/CustomProviderManager.js';
```

ESLint

[error] Type imports must be imported in the same import statement as values, using the type keyword

(rulesdir/inline-type-imports)